recipes/libjxl/all/conanfile.py

```diff
@@ -0,0 +1,103 @@
+from conans import ConanFile, CMake, tools
+import os
+
+required_conan_version = ">=1.33.0"
+
+
+class LibjxlConan(ConanFile):
+    name = "libjxl"
+    description = "JPEG XL image format reference implementation"
+    license = "Apache-2.0"
+    url = "https://github.com/conan-io/conan-center-index"
+    homepage = "https://github.com/libjxl/libjxl"
+    topics = ("image", "jpeg-xl", "jxl", "jpeg")
+    requires = "brotli/1.0.9", "highway/0.12.2", "lcms/2.11"
```

Comment:
Please add these to a proper `requirements` method, calling `self.requires` for each.
When all recipes do this, it's more maintainable for us.

Reply:
Done.